lib/validators/stripped_length_validator.rb

```diff
@@ -2,7 +2,7 @@
 
 class StrippedLengthValidator < ActiveModel::EachValidator
   def self.validate(record, attribute, value, range)
-    if value.nil?
+    if value.blank?
```

Comment:
Should this be tested? Should also probably be in another PR so we keep that PR solely on DB limit changes?

Reply:
Actually this is tested by the shoulda matcher we already have. It seems changing the column type from `text` to `string` makes it test `""` as a value instead of just `nil`. (that’s how I caught this bug in the first place :sweat_smile:)